src/features/Status/StatusPage.tsx

```diff
@@ -96,7 +105,7 @@ class StatusPage extends React.Component<IStatusPageProps, IStatusPageState> {
                     <Button type="button">View/Edit Application</Button>
                   </LinkDuo>
                 </Flex>
-              ) : Date.now() < CONSTANTS.APPLICATION_CLOSE_TIME &&
+              ) : new Date() < new Date(this.state.settings.closeTime) &&
```

Comment:
Since we have an open time, I think this should check if now is between `openTime` and `closeTime`

Reply:
Done